HW1/P2/P2.txt

```diff
@@ -1 +1,7 @@
 # Your discussion here
+
+From the P2a_hist plot, we can see that most partitions are centered at about 1e6 which a few partitions take much larger amount of time. This means partitions take largely different amount of time - over 80% partitions are just doing less than 10% of the work. 
+
```

Comment:
Very good and same idea here! In P2a, we used default partitioning. Just partition all the data into 100 partitions within fixed order, which is fast with the partition time. However, it does not balance the work, since some pixels are concentrated in some certain partitions, such as the light pixels are located in certain partitions, which cause the processing time for those partitions very slow.